pb/api/v1/data.proto

```diff
@@ -162,6 +167,85 @@ message ListRowsResponse {
   }
 }
 
+message GetDashboardTabRequest {
+  string scope = 1;
+  string dashboard = 2;
+  string tab = 3;
+}
+
+message GetDashboardTabResponse {
+
+  // GCSConfig specifies results stored in GCS, typically created by prow.
+  message GCSConfig {
```

Comment:
I think we want to avoid nested messages in general, unless it's only related to the nesting message. Result source messages are debatable, but I think not having it nested is better atm.